src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -512,6 +512,7 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 "BCP082",
                 $"The name \"{name}\" does not exist in the current context. Did you mean \"{suggestedName}\"?",
                 null,
+                null,
                 new CodeFix($"Change \"{name}\" to \"{suggestedName}\"", true, CodeManipulator.Replace(TextSpan, suggestedName)));
```

Comment:
Would you mind updating our existing diagnostics to use the new Uri field? We have a few with links in.